chinese-text-search/setup_run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+rm -rf workspace && \
+python app.py -t index | tee metrics.txt && \
+python app.py -t query <<< 'some text from stdin' | tee >> metrics.txt
```

Comment:
oh I see. You're simulating sending a query via stdin. Then ignore the above comment, if this works well

Reply:
yeah I think it should work, at least metrics.txt looked okay. The app should also exit after reading the string so there should be no blocking here